scylla/src/transport/load_balancing/default.rs

```diff
@@ -1563,4 +1583,315 @@ mod tests {
         }
     }
 
+    impl Default for DefaultPolicy {
+        fn default() -> Self {
+            Self {
+                preferred_datacenter: None,
+                is_token_aware: true,
+                permit_dc_failover: true,
+                latency_awareness: None,
+                pick_predicate: Box::new(Self::is_alive),
+            }
+        }
+    }
```

Comment:
I've put it there on purpose, as it may be tempting to use this default impl by the users, this way losing possibility to e.g. set the preferred datacenter. I thought that the users should always use the builder, thence being aware of the available configuration options.

Reply:
Hmm at first I thought it does't matter where the impl is defined, it doesn't change the visiblity, but because of `#[cfg(test)]` it's only visible to tests. Makes sense :+1: